homeassistant/components/media_player/__init__.py

```diff
@@ -866,6 +866,14 @@ def preload_media_image_url(self, url):
             _async_fetch_image(self.hass, url), self.hass.loop
         ).result()
 
+    def set_last_bad_image_url(self, url):
```

Comment:
Could you explain what difference does it make beyond readability?
Would it change the way it is executed?